docs/Actions/inject-data.md

```diff
@@ -0,0 +1,273 @@
+---
+id: inject-data
+title: Inject Data
+sidebar_label: Inject Data
+---
+
+> Audience: [`Business Users`](/docs/audience#business-users)<br/>
+> Skill Prerequisites: `None`
+
+An action that lets you add data as tokens to the current execution context. Inject data can be used to load data that other actions other actions down the stack need.
+
+## `Typical Use Cases`
+
+- Generate a new token
+- Concatenate 2 tokens
+- Overwrite a token
+
```

Comment:
I saw often the scenario where we use Inject Data to initialize some variables with empty strings so they exist regardless the path the process takes. Is this a valid use case?

Reply:
Yes, that is a valid use case. I forgot about it, but now it's added.